CancelBooking UseCase/CancelBookinng.java

```diff
@@ -4,6 +4,8 @@
 import java.time.LocalDate;
 // Data Layer
 // Represents a reservation with details like customer name, room type, room number, price, CPR, phone number, and booking date.
+
+
```

Comment:
```suggestion
you can make the getter methods more secure by making the field as "final". This ensures that once a reservation is created, it can’t be modified.
```